main.py

```diff
@@ -44,10 +45,14 @@ def closeSpotify():
 def openSpotify(path):
     """Start a Spotify instance"""
     if path is None:
-        path = shutil.which("spotify")
+        path = shutil.which("spotify") 
     if path is None and os.name == "posix":
         path = "/Applications/Spotify.app"
 
+    # Path may still be `None`, if so try to get it from the default installation location
+    if path is None:
+        path = f"C:/{getuser()}/AppData/Roaming/Spotify/Spotify.exe"
```

Comment:
The path is actually wrong...
1. Windows doesn't use forward slashes.
2. Missing "Users" directory.
3. The username is not necessarily the same as the user folder name on Windows.